lib/portlayer/exec/container.go

```diff
@@ -393,16 +393,24 @@ func infraContainers(ctx context.Context, sess *session.Session) ([]mo.VirtualMa
 
 // find the childVM for this resource pool by name
 func childVM(ctx context.Context, sess *session.Session, name string) (*vm.VirtualMachine, error) {
-	searchIndex := object.NewSearchIndex(sess.Client.Client)
-	child, err := searchIndex.FindChild(ctx, VCHConfig.ResourcePool.Reference(), name)
```

Comment:
@dougm Here is the spec to create container VM:
https://github.com/vmware/vic/blob/master/lib/spec/spec.go#L94
While it's vSAN, looks like we can not specify full vmx path. so It's still possible to rename the folder path, right?